tensorflow_lite_support/python/task/vision/pybinds/_pywrap_image_classifier.cc

```diff
@@ -66,43 +63,42 @@ PYBIND11_MODULE(_pywrap_image_classifier, m) {
             options.mutable_class_name_blacklist()->CopyFrom(
                 classification_options.class_name_denylist());
 
-            return ImageClassifier::CreateFromOptions(options);
+            auto classifier = ImageClassifier::CreateFromOptions(options);
+            return core::get_value(classifier);
           })
       .def("classify",
            [](ImageClassifier& self, const ImageData& image_data)
-               -> tflite::support::StatusOr<processor::ClassificationResult> {
-             ASSIGN_OR_RETURN(std::unique_ptr<FrameBuffer> frame_buffer,
-                              CreateFrameBufferFromImageData(image_data));
-             ASSIGN_OR_RETURN(ClassificationResult vision_classification_result,
-                              self.Classify(*frame_buffer));
-
+               -> processor::ClassificationResult {
+             auto frame_buffer = CreateFrameBufferFromImageData(image_data);
+             auto vision_classification_result = self.Classify(
+                     *core::get_value(frame_buffer));
              // Convert from vision::ClassificationResult to
-             // processor::ClassificationResult as required by the Python layer.
+             // processor::ClassificationResult
              processor::ClassificationResult classification_result;
-             classification_result.ParseFromString(
-                 vision_classification_result.SerializeAsString());
+               classification_result.ParseFromString(
+                 core::get_value(vision_classification_result)
+                 .SerializeAsString());
              return classification_result;
            })
       .def("classify",
            [](ImageClassifier& self, const ImageData& image_data,
               const processor::BoundingBox& bounding_box)
-               -> tflite::support::StatusOr<processor::ClassificationResult> {
+               -> processor::ClassificationResult {
              // Convert from processor::BoundingBox to vision::BoundingBox as
-             // the latter is used in the C++ layer.
```

Comment:
FYI there are some typo fix reverted here, as well as above. I will fix them when importing the PR for review but please be more careful next time and pull the latest code from master. The code review happens internally using Google's code management system so sometimes I addressed internal feedback directly, resulting in some difference between the code in the pull request and what is merged into master.

Reply:
@khanhlvg Alright, sorry about that. I'll keep that in mind.